src/Components/LikedVideos.jsx

```diff
@@ -11,40 +11,44 @@ export const LikedVideos = () => {
     return video;

```

Comment:
Here too you can use ES6 syntax and write the function in one line

```suggestion
const getVideo = ( videoURL ) => videos.find(( video ) => video.url === videoURL );
```